Transmission.xcodeproj/project.pbxproj

```diff
@@ -2300,7 +2299,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cd third-party/dht\nln -s . dht";
+			shellScript = "cd third-party/dht\nrm dht\nln -s . dht";
```

Comment:
I believe you'd need to revert that change, or explain it

Reply:
It was actually already addressed in February 2017 (5 years ago):
https://github.com/transmission/transmission/commit/dcc9a4308b328a87d9c216ccf230faa6844a280b
So this change definitely needs to be removed from this pull request.

Reply:
Given that this PR is from Jan 2017 and nothing was done till now, telling me it was addressed in February 2017 might not be the best response? I am no longer working on this after 5 years. Please feel free to close the PR.

Reply:
@FahimF thanks for the reply.

Reply:
My apologies too.
I only started being active in 2022, and there were lots of pending PRs.